.github/workflows/build_and_push.yaml

```diff
@@ -0,0 +1,23 @@
+on:
+  workflow_dispatch:
```

Comment:
if i use dispatch, will it use the git sha instead of tag?

Reply:
branch name / tag
it's possible to use commit SHA if we want to (aka https://stackoverflow.com/questions/65895649/how-to-pre-fill-an-input-field-in-a-workflow-dispatch-github-action-with-github)

Reply:
it is suffiencient